sympy/physics/mechanics/system.py

```diff
@@ -1,8 +1,1009 @@
+
+
+class System(_Methods):
+    """Class to define a system and form the equations of motion.
+
+    Explanation
+    ===========
+
+    A ``System`` instance stores the different objects associated with a model,
+    including bodies, joints, constraints, and other relevant information. With
+    all the relationships between components defined, the ``System`` can be used
+    to form the equations of motion using a backend, such as ``KanesMethod``.
+    The ``System`` has been designed to be compatible with third-party
+    libraries for greater flexibility and integration with other tools.
 
-__all__ = ['SymbolicSystem']
+    Attributes
+    ==========
+
+    origin : Point
+        Global origin of the system.
+    frame : ReferenceFrame
+        Inertial reference frame of the system.
+    x : Vector
+        Unit vector in the x direction of the inertial reference frame.
+    y : Vector
+        Unit vector in the y direction of the inertial reference frame.
+    z : Vector
+        Unit vector in the z direction of the inertial reference frame.
+    q : Matrix
+        Matrix of all the generalized coordinates.
+    u : Matrix
+        Matrix of all the generalized speeds.
+    q_ind : Matrix
+        Matrix of the independent generalized coordinates.
+    q_dep : Matrix
+        Matrix of the dependent generalized coordinates.
+    u_ind : Matrix
+        Matrix of the independent generalized speeds.
+    u_dep : Matrix
+        Matrix of the dependent generalized speeds.
+    kdes : Matrix
+        Matrix of the kinematical differential equations.
+    bodies : tuple of BodyBase subclasses
+        Tuple of all bodies that make up the system.
+    joints : tuple of Joint
+        Tuple of all joints that connect bodies in the system.
+    loads : tuple of LoadBase subclasses
+        Tuple of all loads that have been applied to the system.
+    holonomic_constraints : Matrix
+        Matrix with the holonomic constraints as rows.
+    nonholonomic_constraints : Matrix
+        Matrix with the nonholonomic constraints as rows.
+    eom_method : class
+        Backend for forming the equations of motion.
+
+    Examples
+    ========
+
+    In the example below a cart with a pendulum is created. The cart moves along
+    the x axis of the rail and the pendulum rotates about the z axis. The length
+    of the pendulum is ``l`` with the pendulum represented as a particle. To
+    move the cart a time dependent force ``F`` is applied to the cart.
+
+    We first need to import some functions and create some of our variables.
+
+    >>> from sympy import symbols, simplify
+    >>> from sympy.physics.mechanics import (
+    ...     mechanics_printing, dynamicsymbols, RigidBody, Particle,
+    ...     ReferenceFrame, System, PrismaticJoint, PinJoint)
+    >>> mechanics_printing(pretty_print=False)
+    >>> g, l = symbols('g l')
+    >>> F = dynamicsymbols('F')
+
+    Create bodies. A frame is also attached to the particle, as it will be
```

Comment:
What's the "create bodies" referring to?

Reply:
Okay, reading the next paragraphs I see what you're saying here.

Reply:
Is this clearer:
`The next step is to create bodies. It is also useful to create a frame for locating the particle with respect to the pin joint later on, as a particle does not have a body-fixed frame.`